examples/diagnostics_and_criticism/sampler-stats.ipynb

```diff
@@ -21,22 +21,34 @@
      "name": "stdout",
```

Comment:
<p>Note that not all the parameters defined in ArviZ schema are present nor all the variables in <code>trace.sample_stats</code>  are in ArviZ schema, some are specific to PyMC3. We should not replace the original definitions by the descriptions in arviz shcema but combine them. </p><p> </p><p>There are also some new variables that have no explanation in the old notebook, I think these are related to the extra info @michaelosthege added to the trace report.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/95/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>um that would be weird, combining them I mean, because some variables are like -</p><p> </p><p>step_size_nom in arviz and step_size_bar in pymc3 and so which one should we go for? should we put all the variables, if thats what you meant? also i think the total number of variables and names should be consistent in pymc3 and arviz if we are using arviz for sampler_stats what we can do is, just not use some of the variables in each library when theyre not relevant, but they should be defined in both the libraries? does that make  any sense</p>